src/include/ndpi_private.h

```diff
@@ -168,6 +168,9 @@ struct ndpi_global_context {
 
   /* NDPI_PROTOCOL_MSTEAMS */
   struct ndpi_lru_cache *msteams_global_cache;
+  
+  /* NDPI_FIRST_PKT_CLASSIFICATION_CACHE */
+  struct ndpi_lru_cache *fpc_global_cache;
```

Comment:
Since we are using this cache for DNS mapping, I would prefer `fpc_dns_global_cache` name.
Same change pretty much everywhere....

Reply:
@IvanNardi 
Thank you very much for your kind review and feedback.i'm working on the changes and update you once tested in my local env.